src/pint/models/stand_alone_psr_binaries/DD_model.py

```diff
@@ -772,33 +787,33 @@ def d_delayS_d_par(self, par):
     def delayE(self):
         """Binary Einstein delay
 
-        T. Damour and N. Deruelle(1986)equation [25]
+        T. Damour and N. Deruelle (1986) equation [25]
         """
         sinE = np.sin(self.E())
-        return self.GAMMA
+        return self.GAMMA * sinE
```

Comment:
There's no real reason for the extra temporary line for `sinE`.  I'd recommend just putting that in the return.

Reply:
done.